json-schema/json-schema-generic/src/main/scala/endpoints4s/generic/JsonSchemas.scala

```diff
@@ -74,9 +75,13 @@ trait JsonSchemas extends algebra.JsonSchemas {
       with GenericDescriptions
       with GenericTitles {
 
-    implicit def emptyRecordCase: DocumentedGenericRecord[HNil, HNil] =
+    @deprecated("Replaced by `genericEmptyRecord`", "1.2.0")
+    def emptyRecordCase: DocumentedGenericRecord[HNil, HNil] =
```

Comment:
Existing methods involved in the generic derivation process are not anymore implicit so that there is no risk for the compiler to use them when deriving schemas.
We don’t remove them to keep backward binary compatibility, but we mark them as `@deprecated` to signal that we can remove them in the next major release.